model_info/arwiki.damaging.md

```diff
@@ -1,12 +1,12 @@
 Model Information:
 	 - type: GradientBoosting
 	 - version: 0.5.0
-	 - params: {'population_rates': None, 'min_samples_split': 2, 'scale': True, 'warm_start': False, 'loss': 'deviance', 'labels': [True, False], 'subsample': 1.0, 'learning_rate': 0.01, 'min_impurity_decrease': 0.0, 'init': None, 'validation_fraction': 0.1, 'presort': 'auto', 'center': True, 'min_samples_leaf': 1, 'criterion': 'friedman_mse', 'max_features': 'log2', 'max_leaf_nodes': None, 'multilabel': False, 'label_weights': OrderedDict([(True, 10)]), 'tol': 0.0001, 'max_depth': 3, 'n_estimators': 100, 'min_impurity_split': None, 'n_iter_no_change': None, 'random_state': None, 'min_weight_fraction_leaf': 0.0, 'verbose': 0}
+	 - params: {'labels': [True, False], 'learning_rate': 0.01, 'warm_start': False, 'n_iter_no_change': None, 'init': None, 'multilabel': False, 'ccp_alpha': 0.0, 'min_samples_leaf': 1, 'min_samples_split': 2, 'center': True, 'n_estimators': 100, 'population_rates': None, 'max_depth': 3, 'verbose': 0, 'min_impurity_decrease': 0.0, 'subsample': 1.0, 'max_features': 'log2', 'scale': True, 'label_weights': OrderedDict([(True, 10)]), 'loss': 'deviance', 'tol': 0.0001, 'criterion': 'friedman_mse', 'min_impurity_split': None, 'presort': 'deprecated', 'max_leaf_nodes': None, 'validation_fraction': 0.1, 'min_weight_fraction_leaf': 0.0, 'random_state': None}
 	Environment:
-	 - revscoring_version: '2.5.1'
-	 - platform: 'Linux-4.9.0-9-amd64-x86_64-with-debian-9.9'
+	 - revscoring_version: '2.8.0'
```

Comment:
This is unrelated to the ptwiki model but shouldn't this be 2.8.2?

Reply:
Turns out there were some minor fixes that were unrelated to Arabic language processing that lead to 2.8.2 so I opted not to rebuild a few of those models.